samples/highcharts/demo/organization-drilldown/demo.details

```diff
@@ -0,0 +1,10 @@
+---
+name: Organization chart drilldown
+authors:
+  - Torstein Hønsi
+js_wrap: b
+tags:
+  - Highcharts demo
```

Comment:
I don't think we should tag this as a featured demo. Instead, we should add basic drillown samples (demo/column-drilldown, demo/pie-drilldown plus the new sample) as `@sample` items to the [drilldown main page](https://api.highcharts.com/highcharts/drilldown).